packages/testing/addon/mirage-graphql/index.js

```diff
@@ -45,11 +45,14 @@ export default function createGraphqlHandler(server) {
         SelectedOption: ({ value }) => {
           const option = server.schema.options.findBy({ slug: value });
 
-          return {
-            slug: value,
-            label: option.label,
-            __typename: "SelectedOption",
-          };
+          if (option) {
+            return {
+              slug: value,
+              label: option.label,
+              __typename: "SelectedOption",
+            };
+          }
+          return null;
```

Comment:
cc @anehx

Reply:
Makes sense. I'd put the null return into the if as that should be the "special case" but that's just personal flavour.